src/spotify.c

```diff
@@ -1344,15 +1002,30 @@ static void playlist_update_in_progress(sp_playlist *pl, bool done, void *userda
     {
       DPRINTF(E_DBG, L_SPOTIFY, "Playlist update (status %d): %s\n", done, fptr_sp_playlist_name(pl));
 
-      spotify_playlist_save(pl);
+      if (spotify_access_token_valid)
+        {
```

Comment:
Is the indentation ok here? Here on github the brackets don't seem equally aligned.